src/guide-dist.md

```diff
@@ -114,6 +114,11 @@ much faster than sampling `n` trials individually.
 
 The [`Poisson`] distribution expresses the expected number of events
 occurring within a fixed interval, given that events occur with fixed rate λ.
+[`Poisson`] distribution sampling generates `Float` values because `Float`s
+are used in the sampling calculations, and we prefer to defer to the user on
+integer types and the potentially lossy and panicking associated conversions.
+For example, in Rust >= 1.45, `u64` values can be attained with
+`rng.sample(Poisson) as u64`.
```

Comment:
Looks good, thanks! Maybe we should add something like this: "(In earlier Rust versions this may be result in undefined behavior.)"

Reply:
Thanks @vks!  I've added that elaboration on float-to-int conversions in Rust <1.45 vs. Rust >=1.45, and used the "Note..." pattern from the proceeding section https://github.com/rust-random/book/pull/34/files#diff-ca60f3595d15c2c584e2306702decf13e1cb05fa875731a5ed8c0b20e5bd1bdaR112.  Do you think that looks good?

Reply:
Perfect, thanks!